client/src/configs/env.ts

```diff
@@ -55,3 +55,6 @@ export const DISABLE_LOGIN_PROTECTION = process.env.REACT_APP_DISABLE_LOGIN_PROT
 
 /** Activate the product-catalogue menu */
 export const ENABLE_PRODUCT_CATALOGUE = process.env.REACT_APP_ENABLE_PRODUCT_CATALOGUE === 'true';
+
+/** Activate the plans menu */
+export const ENABLE_PLANS = process.env.REACT_APP_ENABLE_PLANS === 'true';
```

Comment:
Have you added this to the envs doc?

Reply:
Yeah https://github.com/OpenSRP/web/pull/210/files#diff-cd3585e447df7ff668c4e9f8a48fffcdb733de76ef54e27e2162dfb5c02b6398L12